contracts/MarginCalculator.sol

```diff
@@ -2,11 +2,308 @@
  * SPDX-License-Identifier: UNLICENSED
  */
 pragma solidity 0.6.10;
+pragma experimental ABIEncoderV2;
+
+import {Initializable} from "./packages/oz/upgradeability/Initializable.sol";
+import {SafeMath} from "./packages/oz/SafeMath.sol";
+import {OtokenInterface} from "./interfaces/OtokenInterface.sol";
+import {OracleInterface} from "./interfaces/OracleInterface.sol";
+import {AddressBookInterface} from "./interfaces/AddressBookInterface.sol";
+import {FixedPointInt256} from "./libs/FixedPointInt256.sol";
+import {SignedConverter} from "./libs/SignedConverter.sol";
+import {MarginAccount} from "./libs/MarginAccount.sol";
 
 /**
- *
+ * @title MarginCalculator
+ * @author Opyn
+ * @notice Calculator module that check if a given vault is valid.
  */
-// solhint-disable-next-line no-empty-blocks
-contract MarginCalculator {
+contract MarginCalculator is Initializable {
```

Comment:
MarginCalculator is not upgradeable module, why inheriting `Initializable` ?

Reply:
now in the address book it's using updateImpl for calculator, implying that it's upgradable right?

Reply:
Then we should change that in another PR.